packages/strapi-plugin-i18n/admin/src/middlewares.js

```diff
@@ -0,0 +1,21 @@
+const extendCTBInitialDataMiddleware = () => {
```

Comment:
This one is a bit tricky. I need to reflect on that.

My main problem being: every plugin can define a middleware on an action that is part of another plugin.

This means that the plugin intercepting the action needs to know the action names like `ContentTypeBuilder/FormModal/SET_DATA_TO_EDIT`

It means that from one plugin, we need to know
- "plugin specific APIs"
- the action name (of the other plugins) to catch, enhance, and dispatch
- the payload shape passing through the middlewares

Reply:
Yes I agree but I think this issue can be solved by documenting a plugin, what do you think?